<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Usuario</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CREA      USUARIO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/prefijoNuevo/pc.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>81</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PASSWORD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>131</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NOMBRE USUARIO:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CONFIRMA    PASSWORD:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="crear_usuario">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CREAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="volver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt; VOLVER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usuario_nuevo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="ch_check">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ArcadeClassic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MOSTRAR   PASSWORDS</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos_2.qrc"/>
 </resources>
 <connections/>
</ui>
